docs/high/dataset.rst

```diff
@@ -103,6 +103,34 @@ safe to use with very large target selections.  It is supported for the above
    Currently h5py does not support nested compound types, see :issue:`1197` for
    more information.
 
+Multiple indexing
+~~~~~~~~~~~~~~~~~
+
+Indexing a dataset once loads a numpy array into memory.
+If you try to index it twice to write data, you may be surprised that nothing
+seems to have happened:
```

Comment:
Shouldn't these end in a double colon (`::`)

Reply:
Doesn't seem to be needed - I think Spinx recognises the `>>>` markers.

Reply:
Ahh ok